tests/test_property.py

```diff
@@ -268,3 +268,137 @@ def test_to_graph_should_return_subject() -> None:
+    """It returns a belongs_to_module graph isomorphic to spec."""
+    modelproperty = Role()
+    modelproperty.identifier = "http://example.com/properties/1"
+    modelproperty.belongs_to_module = ["core"]
+
+    src = """
+    @prefix dct: <http://purl.org/dc/terms/> .
+    @prefix rdf: <http://www.w3.org/1999/02/22-rdf-syntax-ns#> .
+    @prefix rdfs: <http://www.w3.org/2000/01/rdf-schema#> .
+    @prefix dcat: <http://www.w3.org/ns/dcat#> .
+    @prefix modelldcatno: <https://data.norge.no/vocabulary/modelldcatno#> .
+
+    <http://example.com/properties/1>    a modelldcatno:Role ;
+        modelldcatno:belongsToModule "core";
+    .
+    """
+    g1 = Graph().parse(data=modelproperty.to_rdf(), format="turtle")
+    g2 = Graph().parse(data=src, format="turtle")
+
+    assert_isomorphic(g1, g2)
+
+
+def test_to_graph_should_return_forms_symmetry_with() -> None:
+    """It returns an identifier graph isomorphic to spec."""
+    modelproperty1 = Role()
+    modelproperty1.identifier = "http://example.com/properties/1"
+
+    modelproperty2 = Role()
+    modelproperty2.identifier = "http://example.com/properties/2"
+
+    modelproperty1.forms_symmetry_with = modelproperty2
+
+    src = """
+        @prefix dct: <http://purl.org/dc/terms/> .
+        @prefix rdf: <http://www.w3.org/1999/02/22-rdf-syntax-ns#> .
+        @prefix rdfs: <http://www.w3.org/2000/01/rdf-schema#> .
+        @prefix dcat: <http://www.w3.org/ns/dcat#> .
+        @prefix modelldcatno: <https://data.norge.no/vocabulary/modelldcatno#> .
+        @prefix skos: <http://www.w3.org/2004/02/skos/core#> .
+        @prefix xkos: <http://rdf-vocabulary.ddialliance.org/xkos#> .
+
+
+        <http://example.com/properties/1>
+                a modelldcatno:Role;
+                    modelldcatno:formsSymmetryWith <http://example.com/properties/2> .
+
+        """
+    g1 = Graph().parse(data=modelproperty1.to_rdf(), format="turtle")
+    g2 = Graph().parse(data=src, format="turtle")
+
+    assert_isomorphic(g1, g2)
+
+
+def test_to_graph_should_return_forms_symmetry_with_bnode() -> None:
+    """It returns an identifier graph isomorphic to spec."""
+    modelproperty1 = Role()
+    modelproperty1.identifier = "http://example.com/properties/1"
+
+    modelproperty2 = Role()
+
+    modelproperty1.forms_symmetry_with = modelproperty2
+
+    src = """
+        @prefix dct: <http://purl.org/dc/terms/> .
+        @prefix rdf: <http://www.w3.org/1999/02/22-rdf-syntax-ns#> .
+        @prefix rdfs: <http://www.w3.org/2000/01/rdf-schema#> .
+        @prefix dcat: <http://www.w3.org/ns/dcat#> .
+        @prefix modelldcatno: <https://data.norge.no/vocabulary/modelldcatno#> .
+        @prefix skos: <http://www.w3.org/2004/02/skos/core#> .
+        @prefix xkos: <http://rdf-vocabulary.ddialliance.org/xkos#> .
+
+        <http://example.com/properties/1>
+                a modelldcatno:Role;
+                    modelldcatno:formsSymmetryWith [ a modelldcatno:Role ]  .
```

Comment:
Ser ut som det manger hva identifier i blank node.

Reply:
Hei @azaxarov ! Er det identifier i den blanke noden du tenker på? Blanke noder er nettopp blanke fordi de ikke har identifier. Eller misforstår jeg spørsmålet?

Reply:
Objektet på `modelldcatno:formsSymmetryWith` ser underspesifisert til meg, altså den har kun type. Kanskje det funker for test sin skyld.

Reply:
Ja, det stemmer dette. Men det er slik blanke noder er. De har ikke identifier. Men jeg kan legge på mer "mat" på det, altså felter slik at det vises, i stedet for bare `[ a modelldcatno:Role ] ` hvis det var det som du mente?

Reply:
Da er det lagt på litt bedre eksempel på testen.. ;-)